NBitcoin/Transaction.cs

```diff
@@ -1619,6 +1619,22 @@ public int GetVirtualSize()
 			return (weight + WITNESS_SCALE_FACTOR - 1) / WITNESS_SCALE_FACTOR;
 		}
 
+		/// <summary>
+		/// Sign a specific coin with the given secret
+		/// </summary>
+		/// <param name="keys">Private key</param>
+		/// <param name="coins">Coins to sign</param>
+		public void Sign(BitcoinSecret key, params ICoin[] coins)

```

Comment:
be careful, using params may even bring more confusion because then the parameter could be omitted completely; see https://github.com/MetacoSA/NBitcoin/pull/269 for more info

Reply:
I think @knocte is right, should not be `params`.

Reply:
I'll remove it. The other overload addresses the reader's confusion, too.

Reply:
Actually removed and added another overload that may make sense.